### ISSUE TITLE
Adding an option to exclude temperature sensors from the interior/exterior groups

### ISSUE THREAD

**Issue by syphernl** [jseidl/hass-magic_areas#55]
The temperature setpoint of my thermostat is being considered a temperature in regards to `sensor.interior_temperature`. Since it might not have reached the actual temperature it could change the outcome of the group result.

```
HASS Version | 0.118.3
HASS Type | Home Assistant Core
Magic_Areas version | 1.1.0
```

**Comment by jseidl** [comment]
Needs an `aggregation_exclude` or similar option. Thanks for the feature request!

**Comment by jseidl** [comment]
Interior / Exterior are now Meta areas which have their own `exclude_entities` config option.

Closed by #74, released on [2.0.0](https://github.com/jseidl/hass-magic_areas/releases/tag/v2.0.0).
